src/palette/elements/Input/Input.tests.tsx

```diff
@@ -33,4 +33,10 @@ describe("Input", () => {
 
     expect(value).toEqual("mockStr")
   })
+
+  it.only("Shows an error message when input has an error", () => {
```

Comment:
```suggestion
  it("Shows an error message when input has an error", () => {
```
it.only left by mistake?

Reply:
yes @brainbicycle, thanks for catching that!